meta/sai_meta.cpp

```diff
@@ -4737,3 +4737,154 @@ void meta_sai_on_fdb_event(
         meta_sai_on_fdb_event_single(data[i]);
     }
 }
+
+// FDB FLUSH
+
+sai_status_t meta_sai_flush_fdb_entries(
+        _In_ sai_object_id_t switch_id,
+        _In_ uint32_t attr_count,
+        _In_ const sai_attribute_t *attr_list,
+        _In_ sai_flush_fdb_entries_fn flush_fdb_entries)
+{
+    SWSS_LOG_ENTER();
+
+    if (flush_fdb_entries == NULL)
+    {
+        SWSS_LOG_ERROR("function pointer is NULL");
+
+        return SAI_STATUS_INVALID_PARAMETER;
+    }
+
+    if (attr_count > MAX_LIST_COUNT)
```

Comment:
should at least have one attribute in the list?

Reply:
no, since list can be empty